hashicorp/boundary-ui
# Icu 14403 add dropdown to select and save log verbosity

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14403

# Description
Added card to be able to change log levels of app as well as show where the logs are saved. 

## Screenshots (if appropriate)
<img width="1392" alt="image" src="https://github.com/user-attachments/assets/44d478ee-cf64-4b98-83da-73a3ef2e1232">

<img width="951" alt="image" src="https://github.com/user-attachments/assets/27f99958-e6f1-4cf2-8703-eca67e37098b">


## How to Test
Start desktop client and confirm that changing log levels works. Debug should show search request times. Confirm the log location is correct as well. 

Confirm that log level config is saved correctly, it should be located at: `~/Library/Application Support/Boundary/config.json`

## Checklist
